views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Map.js

```diff
@@ -130,9 +130,9 @@ define([
             interaction = self.widget.element;
 
         interaction.onPci('responseChange', function (latex) {
-            if (!self.inGapMode(self) && self.activeEditId != null) {
+            if (self.inGapMode(self) === false && self.activeEditId != null) {
                 self.correctResponses[self.activeEditId] = latex;
-            } else if (self.inGapMode(self) && self.activeEditId != null) {
+            } else if (self.inGapMode(self) === true && self.activeEditId != null) {
```

Comment:
niptick: by convention

```suggestion
            } else if (self.inGapMode(self) === true && self.activeEditId !== null) {
```